### ISSUE TITLE
Does Ejecta support tvOS 13?

### ISSUE THREAD

**Issue by fedeali90** [phoboslab/Ejecta#672]
Hello,
Is Ejecta still supported in 2019, and does it support tvOS 13?
Thank you in advance.

**Comment by phoboslab** [comment]
No, it's not actively supported anymore. I don't know about tvOS 13 either. It should still work, though.

Also see https://github.com/phoboslab/Ejecta/issues/669#issuecomment-490000775

**Comment by smith-chris** [comment]
I can only tell you that my game built with ejecta still works flawlessly on iOS 13 :D
You can test it yourself: https://apps.apple.com/pl/app/fruitsvsveggies/id1458096711

I hope that'll help you somehow

Cheers

**Comment by fedeali90** [comment]
Your game is nice! :D 
Great to see that Ejecta is still working on iOS 13. I hope it works also for tvOS 13.

**Comment by lianglijin** [comment]
Can this project still run in Tvos？I can't run the demo in the Xcode ( Version 11.4 )
